test/fixtures/teams.yml

```diff
@@ -6,93 +6,107 @@ team_one:
   team_captain: user_one
   eligible: false
   division: high_school
+  users_count: 1
 
 team_two:
   team_name: team_two
   affiliation: school2
   eligible: false
   division: college
+  users_count: 0
 
 team_three:
   team_name: team_three
   affiliation: school3
   team_captain: user_four
   eligible: true
   division: professional
+  users_count: 3
 
 full_team:
   team_name: full_team
   affiliation: school3
   team_captain: full_team_user_one
   eligible: true
   division: high_school
+  users_count: 5
 
 team_with_special_chars:
   team_name: $0m3Thing @mazing
   affiliation: school!
   team_captain: user_on_team_with_special_chars
   eligible: true
   division: high_school
+  users_count: 1
 
 last_team:
   team_name: Last Team
   affiliation: lastschool
   team_captain: user_on_last_team
   eligible: true
   division: high_school
+  users_count: 1
 
 team_four:
   team_name: team_four
   affiliation: school4
   team_captain: user_four
```

Comment:
Actually this may be something @Bialogs needs to get looped in on, I think he initially setup the fixtures, either the team_captain needs to be removed needs to be actually added to the team.